superset-frontend/src/common/components/Tooltip.tsx

```diff
@@ -21,7 +21,11 @@ import { Tooltip as BaseTooltip } from 'src/common/components';
 import { TooltipProps } from 'antd/lib/tooltip';
 
 const Tooltip = (props: TooltipProps) => (
-  <BaseTooltip overlayStyle={{ fontSize: '12px' }} {...props} />
+  <BaseTooltip
+    overlayStyle={{ fontSize: '12px', lineHeight: '1.6' }}
```

Comment:
should probably use the Theme variables here... maybe `useTheme` would do the trick?

Reply:
Good point!